component/main.jsonnet

```diff
@@ -67,14 +67,48 @@ local identityProviders = [
   if idp.type == 'LDAP'
 ];
 
+local ldapSync =
+  local ldapSyncServiceAccount = com.namespaced(params.namespace, kube.ServiceAccount('ldap-sync'));
+
+  [
+    ldapSyncServiceAccount,
+    kube.ClusterRoleBinding('ldap-sync') {
+      subjects_: [{
+        kind: 'ServiceAccount',
+        metadata: {
+          name: ldapSyncServiceAccount.metadata.name,
+          namespace: params.namespace,
+        },
+      }],
+      roleRef_: {
+        kind: 'ClusterRole',
+        metadata: {
+          name: 'cluster-admin',
+        },
+      },
+    },
+  ] + std.flattenArrays([
+    oauth.LdapSync(params.namespace, idp, ldapSyncServiceAccount.metadata.name)
```

Comment:
What's the reason for moving the LdapSync stuff itself into the component library? Is there a use case where some other component wants to create LdapSync objects for itself?